spec/concepts/api/v1/users/confirmations/operation/show_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::V1::Users::Confirmations::Operation::Show do
+  describe '.call' do
+    subject(:operation) { described_class.call(params) }
+
+    let(:user) { create(:user) }
+    let(:user_data) { { id: user.id, email: user.email, created_at: user.created_at } }
+    let(:email_token) { generate_token(user_data) }
+    let(:params) { { email_token: email_token } }
+
+    before { stub_const('Api::V1::Lib::Service::EmailToken::HMAC_SECRET', 'test123456') }
+
+    describe 'Success' do
+      it 'returns updated user' do
```

Comment:
confirmates target user...

expect { operation }.to change { user.reload.confirmed }.from(false).to(true)....